<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Device_info</class>
 <widget class="QFrame" name="Device_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>201</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="Read">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Write">
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>271</width>
     <height>178</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="2">
     <widget class="QLineEdit" name="Check_Date_value"/>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="SW_ver_value"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="SW_ver">
      <property name="text">
       <string>SW Ver.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="PCB_ver">
      <property name="text">
       <string>PCB Ver.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Man_Date">
      <property name="text">
       <string>Manufacturing Date</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="Man_Date_value"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Check_Date">
      <property name="text">
       <string>Check Date</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="Calib_Date_value"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="Name_value">
      <property name="text">
       <string>Turbine Spirometer AEM-1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Calib_Date">
      <property name="text">
       <string>Calibration Date</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="SN">
      <property name="text">
       <string>Serial Number</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="SN_value"/>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="PCB_ver_value"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
